README.md

```diff
@@ -111,8 +111,11 @@ To connect to AWS Keyspaces, the following environment variables need to be set:
 - `AWS_KEYSPACE` - Your AWS Keyspace name.
 - `CASSANDRA_HOST` - Cassandra host (e.g. cassandra.us-west-2.amazonaws.com).
 - `CASSANDRA_PORT` - Cassandra port (e.g. 9142).
-- `AWS_ACCESS_KEY_ID` - Your AWS Access Key ID.
-- `AWS_SECRET_ACCESS_KEY` - Your AWS Secret Access Key.
+- `AWS_WEB_IDENTITY_TOKEN_FILE` - AWS web identity token file.
```

Comment:
We can remove this and AWS_ROLE_ARN from the README because they are in the environment after deployment, so no need to make the user think that it can be configured.

Reply:
to add to the choir - it's always in `/var/run/secrets/eks.amazonaws.com/serviceaccount/token` given aws sts assume role action succeeds.

Reply:
ok, removed.